pythonfmu/tests/test_builder.py

```diff
@@ -0,0 +1,192 @@
+import itertools
+import platform
+import sys
+import tempfile
+import zipfile
+from pathlib import Path
+
+import pytest
+import pythonfmu
+from pythonfmu.builder import FmuBuilder
+
+DEMO = "pythonslave.py"
+
+
+def get_platform():
+    system = platform.system()
+    is_64bits = sys.maxsize > 2 ** 32
+    platforms = {"Windows": "win", "Linux": "linux", "Darwin": "darwin"}
+    return platforms[system] + "64" if is_64bits else "32"
+
+
+lib_extension = ({"Darwin": "so", "Linux": "so", "Windows": "dll"}).get(platform.system(), None)
+
+# TODO test xml
+
+
+def test_zip_content():
+    script_file = Path(__file__).parent / DEMO
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        FmuBuilder.build_FMU(script_file, dest=tmp_dir)
+
+        fmu = Path(tmp_dir) / "PythonSlave.fmu"
+        assert fmu.exists()
+        assert zipfile.is_zipfile(fmu)
+
+        with zipfile.ZipFile(fmu) as files:
+            names = files.namelist()
+
+            assert (
+                len(names) == 15
```

Comment:
Receiving 14, not 15 here. Windows.

Reply:
Seems to be the .dll that is missing..

Reply:
Ja that one is a bit tricky. If you launch the tests in the package directory and you do not have locally the dll in the `resources` folder. Then the test will fail.

This is the reason in the github workflow, the test is executed in a temporary folder on the previously installed package:
```yaml
        python -m pip install . -v
    - name: Run the Tests
      run: |
        mkdir test_dir
        cd test_dir
        python -m pytest --pyargs pythonfmu.tests
```